### PR TITLE
Added Wrangler Workflow and an example to configure workflow prerequisite jobs condition

### PR DESCRIPTION
**Pull request by karanjeets**
Added Wrangler Workflow (pdf) and examples to configure cas workflow prerequisite jobs condition. Linked to #53 and [OODT PR #35](https://github.com/apache/oodt/pull/35)

### CODE REVIEW

File: workflow/src/main/resources/policy/tasks.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 <cas:tasks xmlns:cas="http://oodt.jpl.nasa.gov/1.0/cas">
 
   <task id="urn:drat:RatCodeAudit" name="RadCodeAudit" 
-    class="org.apache.oodt.cas.pge.StdPGETaskInstance">
+    class="org.apache.drat.pge.DRATPGETaskInstance">
```
Comment (chrismattmann) [inline workflow/src/main/resources/policy/tasks.xml]: this should use standard OODT job instance part of 0.10 on. please revert this.

File: workflow/src/main/resources/policy/tasks.xml [hunk 2]
```diff
@@ -35,7 +35,7 @@
   </task>
 
   <task id="urn:drat:MimePartitioner" name="MimePartitioner" 
-    class="org.apache.oodt.cas.pge.StdPGETaskInstance">
+    class="org.apache.drat.pge.DRATPGETaskInstance">
```
Comment (chrismattmann) [inline workflow/src/main/resources/policy/tasks.xml]: this should use standard OODT job instance part of 0.10 on. please revert this.

File: workflow/src/main/resources/policy/tasks.xml [hunk 3]
```diff
@@ -54,10 +54,16 @@
   </task>
 
   <task id="urn:drat:RatAggregator" name="RatAggregator" 
-    class="org.apache.oodt.cas.pge.StdPGETaskInstance">
-        <conditions/>
+    class="org.apache.drat.pge.DRATPGETaskInstance">
```
Comment (chrismattmann) [inline workflow/src/main/resources/policy/tasks.xml]: this should use standard OODT job instance part of 0.10 on. please revert this.

**Comment by chrismattmann** [comment]
@karanjeets fyi my comments

**Comment by karanjeets** [comment]
@chrismattmann I was wondering, I didn't change this part but it seems, I missed #64. Sorry about that. 
The changes are updated in PR. Hopefully, everything is good now.

**Comment by chrismattmann** [comment]
thanks buddy @karanjeets

**Pull request merged**